src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -0,0 +1,159 @@
+                                    key={item.title}
+                                    title={item.title}
+                                    subtitle={item.subtitle}
+                                    shouldPlaceSubtitleBelowSwitch
+                                    wrapperStyle={styles.mv3}
+                                    isActive={item.isActive}
+                                    onToggle={item.onToggle}
+                                    pendingAction={item.pendingAction}
+                                />
+                            ))}
+
+                            <View style={styles.mv3}>
+                                <SpacerView
+                                    shouldShow
+                                    style={[styles.chatItemComposeBoxColor]}
+                                />
+                            </View>
+
+                            <ToggleSettingOptionRow
+                                title={translate('workspace.qbo.advancedConfig.reimbursedReports')}
+                                subtitle={translate('workspace.qbo.advancedConfig.reimbursedReportsDescription')}
+                                shouldPlaceSubtitleBelowSwitch
+                                wrapperStyle={styles.mv3}
+                                // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                                pendingAction={pendingFields?.reimbursementAccountID || pendingFields?.collectionAccountID}
+                                isActive={isSyncReimbursedSwitchOn} // TODO
+                                // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                                onToggle={() =>
+                                    Connections.updatePolicyConnectionConfig(
+                                        policyID,
+                                        CONST.POLICY.CONNECTIONS.NAME.QBO,
+                                        CONST.QUICK_BOOKS_CONFIG.COLLECTION_ACCOUNT_ID,
+                                        isSyncReimbursedSwitchOn ? '' : selectedAccount,
+                                    )
+                                }
+                            />
+
+                            {collectionAccountID && (
+                                <>
+                                    <MenuItemWithTopDescription
+                                        shouldShowRightIcon
+                                        title="Croissant Co Payroll Account" // TODO: set to the current selected value
+                                        description={translate('workspace.qbo.advancedConfig.qboAccount')}
+                                        wrapperStyle={[styles.sectionMenuItemTopDescription]}
+                                        onPress={waitForNavigate(() => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_ACCOUNT_SELECTOR.getRoute(policyID)))}
+                                    />
```

Comment:
This is missing offline indicator, errors and brick road indicator

(same with other menu items)